phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/behavior/ListWidgetSelectorBehavior.java

```diff
@@ -0,0 +1,85 @@
+package org.phoenicis.javafx.components.behavior;
+
+import javafx.event.ActionEvent;
+import javafx.scene.control.ToggleButton;
+import org.phoenicis.javafx.components.control.ListWidgetSelector;
+import org.phoenicis.javafx.components.skin.ListWidgetSelectorSkin;
+import org.phoenicis.javafx.views.common.widgets.lists.ListWidgetType;
+
+import java.util.Optional;
+import java.util.function.Consumer;
+
+/**
+ * The behavior for the {@link ListWidgetSelector} component
+ */
+public class ListWidgetSelectorBehavior
```

Comment:
This is for our miniature list in apps etc., right? I'm asking because the name seems to be a bit generic to me.

Reply:
No, it is the component used to select the list widget, that shows the apps etc.
![grafik](https://user-images.githubusercontent.com/18488086/49182727-50532580-f35b-11e8-86ef-9058a028dcd6.png)

Reply:
Ok, but my point remains: "ListWidget" is somewhat generic given that it refers to this special thing. I must admit that I don't have an idea for a better name currently.

Reply:
Hmm.
Every list widget we have implemented, i.e. ``, `` and ``, all implement the `ListWidget` interface.
So I think the name `ListWidgetSelector` is quite fitting.
What I can change is, that I can add a short reference to `ListWidget` to the `ListWidgetSelector` interface.
What do you think?

Reply:
That's exactly the point. Every list widget **we** have implemented. With the name "ListWidget" it sounds like a very generic list widget (which it is not). In my opinion, the name should make clear that this is about the special list widget in Phoenicis which can display items in 3 different styles.

Reply:
We could extend the component in the future to be able to show an arbitrary number of list widgets, which are somehow passed to the component (most likely via an `ObservableList`).
Then it would be more generic.

Reply:
Yes, I know. Like I said: I have no better name currently. So let's keep it.